apps/api/src/app/rate-limiting/usecases/get-api-rate-limit-maximum/get-api-rate-limit-maximum.spec.ts

```diff
@@ -112,39 +125,78 @@ describe('GetApiRateLimitMaximum', async () => {
       });
     });
 
-    it('should return default api rate limit for the organizations apiServiceLevel when apiServiceLevel IS set on organization', async () => {
+    describe('Organization DOES have api service level specified', () => {
```

Comment:
Organizing the cases for Org DOES/DOES NOT have api service level into `describe` blocks, to reuse the `beforeEach` initializers.